### ISSUE TITLE
jump to definition in a new split pane

### ISSUE THREAD

**Issue by timfeirg** [autozimu/LanguageClient-neovim#508]
Currently the Goto Definition function reuses the current buffer, but often times people might want to open the definition in another split pane and use definition / declaration as documentation for their current work.

Is there an option to enable this behavior? I know I can use something like [`nnoremap <silent> gd :call LanguageClient_textDocument_definition()<CR>:bpr<Cr>:vsplit<Cr>:bn<Cr>`](https://github.com/autozimu/LanguageClient-neovim/issues/35#issuecomment-403734575) to achieve this, but this line certainly wasn't easy to comprehend.

Also, since the above approach does a `vsplit` first, and then `bn` to the previous buffer, user will see the current buffer on the right hand side split pane, and then quickly jump to the target definition / declaration, resulting in a little distracting experience.

**Comment by autozimu** [comment]
Please check this wiki page, https://github.com/autozimu/LanguageClient-neovim/wiki/Goto-definition-in-a-split

**Comment by timfeirg** [comment]
Awesome work! I tried `:call LanguageClient#textDocument_definition({'gotoCmd': 'vsplit'})<CR>` inside vim and indeed it works as expected

However, after I `nnoremap <silent> gd :call LanguageClient#textDocument_definition({'gotoCmd': 'vsplit'})<CR>`, the `gd` command still behaves the same, as if the `{'gotoCmd': 'vsplit'}` doesn't have any effect.

How can I properly map this command? I really can't see what I did wrong in the above `nnoremap` command.
